src/components/Markdown/index.js

```diff
@@ -0,0 +1,7 @@
+import ReactMarkdown from 'react-markdown'
+
+import React from 'react'
+
+export default ({children}) => (
```

Comment:
Here is another example of how I wrapped `react-tabs` with my `Tabs` component:
https://github.com/eggheadio/egghead-instructor-center/blob/master/src/components/Tabs/index.js

And then using it:
https://github.com/eggheadio/egghead-instructor-center/blob/master/src/components/LessonListsByStates/index.js#L17-L72